frontend/packages/dev-console/src/components/pipelines/resource-overview/ResourceLinkList.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { ResourceLink } from '@console/internal/components/utils';
+import './ResourceLinkList.scss';
+
+type ResourceLinkListProps = {
```

Comment:
Maybe we change the API of this to make it easier to call.

Model, "links" & namespace is all we really need.

Reply:
In ResourceLink component `kind` determines which badge to use, so it is also required.

Reply:
Not sure I understand what the concern is... you can compute the values with the ModelKind:

```
// calling file
<ResourceLinkList model={TriggerBindingModel} ... />

// ResourceLinkList
const title = model.labelPlural;
const kind = referenceForModel(model);
```

Reply:
Updated the api to make it easier to call